datacentral.py

```diff
@@ -302,15 +334,39 @@ def generate(offline=False,
                 log.warn("%s: No local cache, skipping." % name)
                 continue
             else:
-                log.info("%s: New repo, cloning." % name)
-                repo = git.Repo.clone_from(url, dir_name)
-                updated = True
+                if url.endswith(".git"):
+                    # Handle GIT Repository URL
+                    log.info("%s: New repo, cloning." % name)
+                    try:
+                        repo = git.Repo.clone_from(url, dir_name)
+                        # For faster checkouts, one file at a time:
+                        #repo = git.Repo.clone_from(url, dir_name, n=True, depth=1)
+                        #repo.git.checkout("HEAD", "datapackage.json")
```

Comment:
I guess you want to get rid of these commented out lines or add an option for "fast cloning" where you omit the history.